numpy/core/src/multiarray/common.h

```diff
@@ -295,8 +295,14 @@ npy_memchr(char * haystack, char needle,
 /*
  * Convert NumPy stride to BLAS stride. Returns 0 if conversion cannot be done
  * (BLAS won't handle negative or zero strides the way we want).
+ *
+ * Return type same as CBLAS_INT (from npy_cblas.h; but we don't #include it here)
```

Comment:
Can we just move `blas_stride` to that file? Any caller including this file without also including `npy_cblas` probably also has the wrong return type.